src/test/java/core/basesyntax/strategy/OperationStrategyTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationStrategyTest {
+    private static final String BALANCE = "b";
+    private static final String PURCHASE = "p";
+    private static final String RETURN = "r";
+    private static final String SUPPLY = "s";
+    private Map<String, OperationHandler> map = new HashMap<>();
+    private OperationStrategy operationStrategy;
+
+    @Before
+    public void setUp() {
+        operationStrategy = new OperationStrategy(map);
+    }
+
+    @Test
+    public void getOperationEmptyTest_Ok() {
+        Set<String> expected = new HashSet<>();
+        expected.add(null);
+        OperationHandler actualHandler = operationStrategy.getByOperation(null);
+        map.put(null, actualHandler);
+        Set<String> actual = map.keySet();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getByOperationBalanceTest_OK() {
+        Set<String> expected = new HashSet<>();
+        expected.add("b");
+        OperationHandler actualHandler = operationStrategy.getByOperation(BALANCE);
+        map.put(BALANCE, actualHandler);
+        Set<String> actual = map.keySet();
+        assertEquals(expected, actual);
```

Comment:
let's better fill map with handlers in BeforeClass

and in tests - call operationStrategy.getByOperation(BALANCE) and check class of the returned object